addons/project/views/project_portal_templates.xml

```diff
@@ -97,31 +97,30 @@
     <template id="portal_tasks_list" name="Tasks List">
         <t t-if="grouped_tasks">
             <t t-call="portal.portal_table">
+                <thead>
+                    <tr>
+                        <th class="text-left">Ref</th>
+                        <th t-if="groupby != 'priority'">Priority</th>
+                        <th>Name</th>
+                        <th name="project_portal_assignees">Assignees</th>
+                        <th t-if="groupby != 'status'">Status</th>
+                        <th t-if="groupby != 'project'">Project</th>
+                        <th t-if="groupby != 'stage'">Stage</th>
+                    </tr>
+                </thead>
                 <t t-foreach="grouped_tasks" t-as="tasks">
                     <thead>
```

Comment:
I think there is something wrong here, isn't it?
This will generate something like this:
```html
<thead>
<!-- len(grouped_tasks) times what's between here -->
<thead>
<tbody>
<!-- and here ... -->
```
So you would get two `thead` for the first `tbody` which is not valid.
This article may help you in order to solve this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody#multiple_bodies

Reply:
Hello @IT-Ideas, changes are done. Thanks.